common/csplit.clip

```diff
@@ -0,0 +1,27 @@
+# csplit
+
+> Split a file into pieces
+> See also: head, tail, split
+> Help: --help
+> Version: --version
+> More information: https://www.gnu.org/software/coreutils/manual/html_node/csplit-invocation.html
+
+- Split a file at specific lines:
+
+`csplit {/?file file: ~/.bashrc} {int+ line: 5}`
+
+- Split a file by a specific line count if it's divisible by this count:
```

Comment:
```suggestion
- Split a file by a specific line count if it's divisible by a specific line count:
```
How about using this uniformly across the pages.

Reply:
I don't like two `specific` terms here. ;)

Reply:
Please check the updated description version.